modules/nf-core/proteinortho/tests/main.nf.test

```diff
@@ -0,0 +1,41 @@
+nextflow_process {
+
+    name "Test Process PROTEINORTHO"
+    script "../main.nf"
+    process "PROTEINORTHO"
+    tag "proteinortho"
+    tag "modules"
+    tag "modules_nfcore"
+
+    test("Should run without failures") {
+
+        when {
+            params {
+                outdir = "$outputDir"
+            }
+            process {
+                file_a = file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true)
+                file_a.copyTo("${workDir}/tmp/prot1.fasta")
+                file_a = file("${workDir}/tmp/prot1.fasta", checkIfExists:true)
+                file_b = file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true)
+
+                """
```

Comment:
```suggestion
                """
                file_a = file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true)
                file_a.copyTo("${workDir}/tmp/prot1.fasta")
                file_a = file("${workDir}/tmp/prot1.fasta", checkIfExists:true)
                file_b = file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true)
```
This should fix the error. `nf-test` injects the code between the quotes inside of a mock script. These files should be inside of that to access the parameters

Reply:
This leads to:


```
Found 1 files in test directory.                                                                                                                                                                        │
│                                                                                                                                                                                                         │
│ Test Process PROTEINORTHO                                                                                                                                                                               │
│                                                                                                                                                                                                         │
│   Test [11f174ae] 'Should run without failures' FAILED (3.131s)                                                                                                                                         │
│                                                                                                                                                                                                         │
│   Assertion failed:                                                                                                                                                                                     │
│                                                                                                                                                                                                         │
│   2 of 2 assertions failed                                                                                                                                                                              │
│                                                                                                                                                                                                         │
│   Nextflow stdout:                                                                                                                                                                                      │
│                                                                                                                                                                                                         │
│   ERROR ~ Script compilation error                                                                                                                                                                      │
│   - file : /home/paul/Documents/proteinortho_dev/REPOSITORIES/modules/.nf-test/tests/11f174aefc8513c6acc47c4259618a31/meta/mock.nf                                                                      │
│   - cause: Unexpected input: '{' @ line 23, column 10.                                                                                                                                                  │
│      workflow {                                                                                                                                                                                         │
│               ^                                                                                                                                                                                         │
│                                                                                                                                                                                                         │
│   1 error                                                                                                                                                                                               │
│                                                                                                                                                                                                         │
│   NOTE: If this is the beginning of a process or workflow, there may be a syntax error in the body, such as a missing or extra comma, for which a more specific error message could not be produced.    │
│                                                                                                                                                                                                         │
│    -- Check '/home/paul/Documents/proteinortho_dev/REPOSITORIES/modules/.nf-test/tests/11f174aefc8513c6acc47c4259618a31/meta/nextflow.log' file for details                                             │
│   Nextflow stderr:                                                                                                                                                                                      │
│                                                                                                                                                                                                         │
│                                                                                                                                                                                                         │
│                                                                                                                                                                                                         │
│                                                                                                                                                                                                         │
│                                                                                                                                                                                                         │
│ FAILURE: Executed 1 tests in 3.134s (1 failed)     
```

looking into the mock.nf, the input looks weird:

```
....
28     // process mapping
29     def input = []
30
31                 file_a = file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true)
32                 file_a.copyTo("/home/paul/Documents/proteinortho_dev/REPOSITORIES/modules/.nf-test/tests/11f174aefc8513c6acc47c4259618a31/work/tmp/prot1.fasta")
33                 file_a = file("/home/paul/Documents/proteinortho_dev/REPOSITORIES/modules/.nf-test/tests/11f174aefc8513c6acc47c4259618a31/work/tmp/prot1.fasta", checkIfExists:true)
34                 file_b = file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true)
35                 input[0] = [
36                     [ id:'test' ], // meta map
37                     [
38                         file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true).renameTo("prot1.fasta")
39                         file_b,
40                     ]
41                 ]
42
```